source/neuropod/bindings/java/src/test/java/com/uber/neuropod/NeuropodTest.java

```diff
@@ -201,6 +196,22 @@ public void InputLongTensor() {
        allocator.close();
     }
 
+    @Test
+    public void InputStringTensor() {
+       NeuropodTensorAllocator allocator = model.getTensorAllocator();
+
+       TensorType type = TensorType.STRING_TENSOR;
+
+       // Note that for String Tensor we can't use low level allocator's method tensorFromMemory
```

Comment:
You may see, we can't allocate String Tensor yet and this is why I can't cover other methods from this review. I don't want to use some hacks and instead next PR will bring necessary functionality and test.